app/script_variant.py

```diff
@@ -0,0 +1,68 @@
+import jautils
+
+from unidecode import unidecode
+
+import logging
+import re
+
+def change_word_to_alphabet(word):
+    """
+    Changes word to alphabet.
+    This method should be called in translate_languages_to_roman().
```

Comment:
Why should the function be called in translate_languages_to_roman()? If it's really the case, you might want to consider inlining this function in translate_languages_to_roman(), so that it can not be called from anywhere else.